packages/css-jss/package.json

```diff
@@ -17,6 +17,7 @@
     "url": "https://github.com/cssinjs/jss/issues/new"
   },
   "files": [
+    "!*.test.js",
```

Comment:
files is actually defining what should be in the package, we don't need to define what should not be in the package in addition

Reply:
oh you are talking about the tests which are next to the src files

Reply:
I wonder as well if that works, because it's a negative match, so it will probably just do nothing and move on to src and still match everything

Reply:
I tried with npm pack and thought it was working but just double-checked and I'm not so sure anymore...

I'll fiddle with it a bit more.

Reply:
otherwise I only see .npmignore as an option

Reply:
I was able to glob ignore specifically the files that end in .test.js that are in the src folders. I could setup .npmignore instead if you think that's a cleaner approach.

Reply:
Pretty confident it's in a good state now. [Here](https://www.digitalocean.com/community/tools/glob?comments=true&glob=src%2F%2A%2A%2F%21%28%2A.test.%2A%29&matches=false&tests=src%2Findex.js&tests=src%2Findex.test.js&tests=src%2Findex.test.tsx&tests=src%2Fwhatever.md&tests=src%2Fhi%2Findex.test&tests=src%2Fhi%2Findex.test.other&tests=src%2Futils%2Ftest.js&tests=src%2Futils%2Ftest.test.js&tests=src%2Futils%2Fother.js) are the cases I was thinking about.

Reply:
yeah, that glob looks scary, at this point I think .npmignore is cleaner

Reply:
I agree it's a little scary 😬

Apparently [files included with the “package.json#files” field cannot be excluded through .npmignore or .gitignore](https://docs.npmjs.com/files/package.json). I wasn't sure if you meant to pull the `files` out of the `package.json` so I kept looking for a better solution.

I found someone suggested doing what I was doing originally so I went back to looking into it. Apparently what I did works, but you have to do it in a specific order. Let me know if that's an ok solution.


Also that link I included above mentions that files called `LICENSE` should be included no matter what you do, so I think it could be taken out of the array. For now, I left it in just in case there was some reason for it to be there.